Sources/NostrSDK/Events/SetMetadataEvent.swift

```diff
@@ -13,6 +13,9 @@ public struct UserMetadata: Codable {
     /// The user's name.
     public let name: String?
     
+    /// The user's display name.
```

Comment:
I think supporting out-of-spec attributes is something we just have to accept as part of an open protocol, but I would like us to note that fact in the DocC comments whenever we do this. That way, client developers will be cautious about depending on such properties.

```suggestion
    /// The user's display name.
    /// > Warning: This property is not part of the Nostr specifications.
```

You could also use `Note` if you feel that `Warning` is too strong.

Reply:
I think warning is OK for now, if someone goes to look at that documentation it should be clear that it is not supported.